dataland-frontend/src/components/resources/taxonomy/search/EuTaxoSearchResults.vue

```diff
@@ -87,7 +90,16 @@ export default {
     },
     scrollToTop(){
       window.scrollTo(0, 0);
-    }
+    },
+    goToData(event){
+      const company=event.data.companyId;
+      this.$router.push('/companies/' + company + '/eutaxonomies')
+    },
   },
 }
-</script>
\ No newline at end of file
+</script>
+<style>
+.table-cursor tbody tr:hover {
+  cursor: pointer;
```

Comment:
Sollen wir die Zeile auch noch sonst wie optisch verändern? Außer dem Cursor - also z.b. etwas dunkleren background oder so?

Reply:
Hovereffekt hinzugefügt

Reply:
Verstehe noch nicht warum das der Fall ist. Der File EuroTaxoSearchResult ist bei mir lokal inhaltsgleich mit dem im Repo und in beiden ist der <style> für den hover Effekt enthalten.

Reply:
ok - passt.